test/e2e_node/node_perf_test.go

```diff
@@ -111,6 +113,21 @@ var _ = SIGDescribe("Node Performance Testing [Serial] [Slow] [Flaky]", func() {
 		framework.Logf("Time to complete workload %s: %v", wl.Name(), perf)
 	}
 
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("ensure environment has enough CPU + Memory to run")
+		minimumRequiredCPU := resource.MustParse("15")
+		minimumRequiredMemory := resource.MustParse("48Gi")
+		localNodeCap := getLocalNode(f).Status.Capacity
```

Comment:
shouldn't this be `Allocatable` not `Capacity`

Reply:
I don't think it makes much difference, but it does seem slightly more correct. Updating.